src/views/LinkMultipleFilesModal.vue

```diff
@@ -2,9 +2,48 @@
 	<div class="multiple-link-modal-container">
 		<NcModal
 			v-if="show"
+			:can-close="isError"
 			@close="closeRequestModal">
 			<div class="multiple-link-modal-content">
 				<LoadingIcon v-if="isLoading" class="loading-spinner" :size="60" />
+				<div v-else-if="chunkingInformation !== null" class="link-progress-information-wrapper">
+					<div v-if="getError" class="link-progress-information-failed">
+						<div class="link-progress-information-failed--details">
+							<div class="link-progress-information-failed--details--info">
+								<AlertCircleOutline fill-color="#e9322d" :size="70" />
```

Comment:
Can we use the variable of the colours provided by the nextcloud here as those colours are handeled by NC itself in different themes?
```suggestion
								<AlertCircleOutline fill-color="--var(red-color-provided-by-nextcloud)" :size="70" />
```